mixer/mixer_test.go

```diff
@@ -0,0 +1,183 @@
+package mixer
+
+import (
+	"os"
+	"testing"
+
+	"github.com/NIHERASE/voxcox/compressor"
+
+	"github.com/go-audio/audio"
+	"github.com/go-audio/wav"
+)
+
+func TestOneChanelInput(t *testing.T) {
+	ch1, ch2, exit2, output := chanelInitializer()
+
+	m := new(Mixer)
+	compressor := new(compressor.MockCompressorImpl)
+	m.audioCompressor = compressor
+
+	buffer1 := getBufferFromFile("../samples/Telephone prompt poss.wav", t)
+
+	go saveChanToFile(output, exit2, buffer1, t)
```

Comment:
This looks more like an example than a test since we have to check the file manually. 
Maybe it should use a real compressor and be in `./examples`.
Here should be a simple test with bare values like in Compressor.